os/drivers/pm/pm.c

```diff
@@ -73,6 +73,7 @@ static ssize_t pm_write(FAR struct file *filep, FAR const char *buffer, size_t l
  *   PMIOC_SLEEP - to make board sleep for given time
  *   PMIOC_TIMEDSUSPEND - to suspend a pm state for given time duration
  *   PMIOC_TUNEFREQ - for changing the operating frequency of the core to save power
+ *   PMIOC_SUSPEND_COUNT - to get suspend count of pm domain
```

Comment:
Can you add command description for PMIOC_DOMAIN_REGISTER and PMIOC_METRICS

Reply:
Done